apps/frontend/src/libs/components/table/table.tsx

```diff
@@ -44,6 +48,17 @@ const Table = <T extends object>({
 	const hasData = data.length !== EMPTY_LENGTH;
 	const isRowSelectable = typeof onRowSelect === "function";
 
+	const tableBodyReference = useRef<HTMLTableSectionElement | null>(null);
+	const [isScrollPresent, setIsScrollPresent] = useState<boolean>(false);
+
+	useEffect(() => {
+		const tableBody = tableBodyReference.current;
+
+		if (tableBody) {
+			setIsScrollPresent(tableBody.scrollHeight > TABLE_BODY_MAX_HEIGHT);
+		}
+	}, [data, isMenuOpened]);
```

Comment:
I think we don't need to calculate all of this, also the table doesn't know about menus or anything specific. Instead, let's just pass isScrollDisabled to the table, also there is no need to check if the scroll was present before or not. This complicates the logic to much, it's ok if we remove padding in case we implement this option, I think it's not that critical.